phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
Sometimes, e.g. viewing a segment of the data during a particular time
window (e.g. trial), or computing an event-triggered average of the LFP for
an infrequent event. But then sometimes you'd want to compute a power
spectrum across many segments of data, or coherence between two LFP traces
across the whole data, or spike-triggered average (which would involve the
whole data trace assuming spikes are high enough rate).

On Tue, Jun 30, 2015 at 7:16 PM, Cyrille Rossant <notifications@github.com>
wrote:

> In phy/cluster/algorithms.py
> <https://github.com/kwikteam/phy/pull/448#discussion_r33605238>:
>
> > @@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
> >      def _create_filter(self):
> >          rate = self._kwargs['sample_rate']
> >          low = self._kwargs['filter_low']
>
> @nsteinme <https://github.com/nsteinme> what analyses do you plan to do
> on LFP data? can these analyses be done chunk by chunk?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kwikteam/phy/pull/448/files#r33605238>.
>